brave-jdbc/src/main/java/com/github/kristofa/brave/jdbc/BraveJdbcListener.java

```diff
@@ -0,0 +1,152 @@
+package com.github.kristofa.brave.jdbc;
+
+import com.github.kristofa.brave.ClientTracer;
+import com.p6spy.engine.common.PreparedStatementInformation;
+import com.p6spy.engine.common.StatementInformation;
+import com.p6spy.engine.event.JdbcEventListener;
+
+import zipkin.Constants;
+import zipkin.TraceKeys;
+
+import java.net.Inet4Address;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.nio.ByteBuffer;
+import java.sql.SQLException;
+
+public final class BraveJdbcListener extends JdbcEventListener {
```

Comment:
I did try this, but that interface doesn't appear to work correctly for insert statements.  The problem appears to be that the `String` of SQL that is passed in when an insert is executed seems to get lost in `SimpleJdbcEventListener` as it is not set on the `StatementInformation`